backend/entityservice/init-db-schema.sql

```diff
@@ -128,19 +130,67 @@ CREATE TABLE bloomingdata (
   -- The receipt token for this data
   token CHAR(48)    NOT NULL UNIQUE,
 
-  -- Store the raw CLK data in a file
+  -- Filename for the raw unprocessed uploaded data
   file  CHAR(64)    NOT NULL,
 
   state PROCESSEDSTATE NOT NULL,
 
-  -- Size in bytes of the uploaded encoding
+  -- Size in bytes of the uploaded encodings
   encoding_size  INT    NULL,
 
-  -- Number of uploaded entries
-  count  INT         NOT NULL
+  -- Number of uploaded encodings
+  count  INT         NOT NULL,
+
+  -- Number of blocks uploaded
+  block_count INT   NOT NULL DEFAULT 1
```

Comment:
both `count` and `block_count` are redundant info in the database. These values are also present int the `blocks` and `encodings` table.
We should be clearer on why we also need those values. What's their meaning?

Reply:
They are a bit redundant - they can be computed by the database easily enough. 

I think the `count` of uploaded encodings is only used in scheduling/chunking the comparison tasks - the block_count won't be used anywhere yet (well it is notionally checked in the encoding_upload task).